packages/core/src/utils/arrayify.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Wraps the value within an array if it is not an array.
+ */
+export function arrayify(value: unknown): any[] {
+  return Array.isArray(value) ? value : [value];
+}
```

Comment:
I'm not a fan of the name arrayify but I don't have a good alternatve either... maybe toArray?

also, here is a narrower type that might work
```suggestion
export function arrayify<T>(value: T | T[]): T[] {
  return Array.isArray(value) ? value : [value];
}
```

Reply:
love this suggestion ty! Updated